cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -46,6 +46,87 @@
 from cleanlab_studio.internal.types import TLMQualityPreset
 
 
+def handle_tlm_exceptions(
```

Comment:
Would it make more sense to move this (+ the `_handle_exception` function) to some utils file?

Reply:
Ah okay, let's just move these functions to the bottom of the file then? (I think it'll be better to have the main TLM class at the top)

Reply:
have also tried that but i believe decorators needs to be defined before the usage otherwise it will complain for "not defined"